guides/v2.1/mtf/mtf_entities/mtf_constraint.md

```diff
@@ -75,10 +75,8 @@ Constraints are performed in order they listed in the data set. However, you can
 
 A test can contain constraints from different modules.
 
-<div class="bs-callout bs-callout-warning" markdown="1">
-  
+{:.bs-callout .bs-callout-warning}
```

Comment:
I _think_ you need to remove the space between the inline attribute and note text, otherwise the note won't render properly.

Reply:
I'm not sure what you mean. There is no space between the inline attribute(Line 78) and note text(Line 79).

Reply:
My mistake. I left this comment in the wrong place apparently... See `guides/v2.3/architecture/tech-stack.md`